scripts/just.config.ts

```diff
@@ -101,7 +101,13 @@ export function preset() {
     condition('jest', () => fs.existsSync(path.join(process.cwd(), 'jest.config.js'))),
   );
 
-  task('lint', parallel('lint-imports', 'eslint'));
+  task(
+    'lint',
+    parallel(
+      condition('lint-imports', () => !isConvergedPackage()),
+      'eslint',
```

Comment:
as a followup, we should remove this completely from v9 and run raw eslint instead. WDYT ?

Reply:
yeah that makes sense 👍